cmd/controller/controller-self-hosted.yaml

```diff
@@ -12,6 +12,8 @@ metricsProvider:
 server:
   host: "0.0.0.0"
   port: 8083
+workerServer:
+  port: 3000
```

Comment:
I think it would be good if we could list the different ports we're exposing and what they are used for. It's not clear to me from the names what the differences are between `server` and `workerServer`.

I think `metricsProvider` is pretty self-explanatory :)

Reply:
A comment would help a lot, a better name might help even more.

Reply:
Added comment in config file to clarify. The reason for using workerServer was it's description here: https://github.com/radius-project/radius/blob/0077cb8ca4a39ea8bca0d182bfe3f19238a05626/pkg/armrpc/hostoptions/providerconfig.go#L66 , so I thought I could use it.
Let me know if this is okay. Else, I will extend ProviderConfig struct and add, say, a HealthProviderOptions struct and use it instead of WorkerServerOptions.

Reply:
I think this is fine based on the comment. Thanks.